backend/Tasque.BLL/MappingProfiles/BoardProfiles.cs

```diff
@@ -1,15 +1,12 @@
 ﻿using AutoMapper;
-using Tasque.Core.Common.DTO.Board;
-using Tasque.Core.Common.DTO.Organization;
-using Tasque.Core.Common.Entities;
 
 namespace Tasque.Core.BLL.MappingProfiles
 {
     public class BoardProfiles : Profile
     {
         public BoardProfiles()
         {
-            CreateMap<Board, BoardDto>().ReverseMap();
+
         }
     }
 }
```

Comment:
You can remove it if there is nothing in it

Reply:
fixed